searx/answerers/models.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from typing import TypedDict, Tuple, List
+import abc
+from searx.search.models import SearchQuery
+
+
+class AnswerDict(TypedDict):
+    answer: str
+
+
+class AnswerSelfInfoDict(TypedDict):
+    name: str
+    description: str
+    examples: List[str]
+
+
+class AnswerModule(metaclass=abc.ABCMeta):
```

Comment:
Should add docstrings to this module class, but will need to scour current documentation / code to figure out a good docstring.

Reply:
Just write what you think s good to know .. I can improve the docstrings in my review.

Reply:
Done - I see its used for autocomplete, so added docs around that.